pkg/services/featuremgmt/registry.go

```diff
@@ -829,7 +829,7 @@ var (
 			FrontendOnly:    false,
 			Owner:           grafanaAlertingSquad,
 			RequiresRestart: true,
-			Enabled:         true,
+			Expression:      "true", // enabled by default
```

Comment:
^^^ alertingNoDataErrorExecution

Reply:
@grafana/alerting-squad -- is the intention with "enabled" that it would be enabled by default?

Reply:
That seems to have been the intention; https://github.com/grafana/grafana/pull/77242

/cc @yuri-tceretian

Reply:
Yes. The intention is to keep the feature flag in case users need to disable it. I plan to delete it after 10.3 is cut.